docker-compose.yml

```diff
@@ -68,13 +67,15 @@ services:
     environment:
       APP_INSTALLATION: "opensource"
       APP_ENV: "prod"
-      POSTGRES_DSN_LIST: "postgres://us:us@pg-us:5432/us-db-ci_purgeable"
+      POSTGRES_DSN_LIST: ${US_POSTGRES_DSN_LIST:-postgres://us:us@pg-us:5432/us-db-ci_purgeable}
       MASTER_TOKEN: "fake-us-master-token"
       PG_TEST_DATA: 1
       CH_DEMO_DATA: 1
       HC: ${HC}
     ports:
       - "50500:80"
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
```

Comment:
`host.docker.internal` is an alias for `localhost` on docker host machine. For cases when db is located there.